gitops/kcp/base/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - https://raw.githubusercontent.com/openshift-pipelines/pipelines-service/main/gitops/triggers/triggers-crds/base/namespace.yaml
```

Comment:
I see just the triggers CRDs. What about pipelines CRDs?

Reply:
That's a mistake. I should have moved the resource locally.

This does instanciate the CRDs. The latest revision should make more sense.